dimagi/pages/static/style/pages/open_source/open_source_home.scss

```diff
@@ -0,0 +1,47 @@
+@import 'app/mixins/background';
+
+
+.about-image {
+  $_base-height: 150px;
+  $_height-by-size: (
+    small: $_base-height,
+    medium: $_base-height,
+    large: $_base-height,
+    extra-large: $_base-height,
+  );
+  @include make-bg(image, '../../../pages/home/build', $_base-height, $_height-by-size, '.jpg');
+}
+
+#oss-logos {
+  display: flex;
+  align-items: center;
+}
+
+.bsd-image {
+  background-position: center;
+  background-repeat: no-repeat;
+  background-image: url('../../../pages/open_source/open_source_home/bsd-logo.png');
+  position: relative;
+  height: 75px;
+  width: 100%;
+  background-size: contain;
+}
+
+.asf-image {
+  background-position: center;
+  background-repeat: no-repeat;
+  background-image: url('../../../pages/open_source/open_source_home/apache-software-foundation-logo.png');
```

Comment:
@biyeun can you review this if you have a moment? I tried to get it working with `background-cover` but I couldn't figure out how (but seemed like everything else similar was using that). do i need to have a thumbnail file in order to make that work?